pkg/credentials/credentials.go

```diff
@@ -156,6 +156,16 @@ func AppendCloudProviderVolumes(dpa *oadpv1alpha1.DataProtectionApplication, ds
 				continue
 			}
 
+			pluginNeedsCheck, foundInBSLorVSL := providerNeedsDefaultCreds[string(plugin)]
```

Comment:
Do we have to make sure to strip out velero.io here?

Reply:
We do not, this gets stripped in `noDefaultCredentials` func